### PR TITLE
refs #1226 respect the WSDL's declaration of the SOAPAction intent, a…

### PR DESCRIPTION
**Pull request by davidnich**
…dded SoapClient::callOperation() for more control over the messages generated with the SoapClient including the possibility to override the SOAPAction value if necessary

### CODE REVIEW

File: RELEASE-NOTES [hunk 1]
```diff
@@ -1,4 +1,19 @@
 *************
+version 1.3.2
+*************
+
+* WSDL module updates:
+  + supress emitting a SOAPAction header in requests if the binding gives an
```
Comment (tethal) [inline RELEASE-NOTES]: typo in suppress (also in docs/mainpage.doxygen.tmpl and WSDL.qm)

**Comment by davidnich** [comment]
@tethal wait - do not merge yet please

**Comment by davidnich** [comment]
@tethal please review & merge when you have a minute

**Comment by pavelkveton** [comment]
I'm fine with it - let's wait for Ondra at least until we are ready to create the release...

**Pull request merged**